server/webhook.go

```diff
@@ -310,13 +290,47 @@ func (s *Server) createSchemaUpdateIssue(ctx context.Context, repository *api.Re
 
 	issue, err := s.createIssue(ctx, issueCreate, api.SystemBotID)
 	if err != nil {
-		s.l.Warn("Failed to create update schema task for added repository file", zap.Error(err),
+		s.l.Warn("Failed to create update schema issue for added repository file", zap.Error(err),
 			zap.String("file", added))
 		return nil, nil
 	}
 	return issue, nil
 }
 
+func (s *Server) createTenantSchemaUpdateIssue(ctx context.Context, repository *api.Repository, mi *db.MigrationInfo, vcsPushEvent vcs.VCSPushEvent, commit gitlab.WebhookCommit, added string, statement string) (*api.Issue, error) {
+	if mi.Environment != "" {
+		return nil, fmt.Errorf("environment isn't accepted in schema update for tenant mode project")
+	}
+
+	m := &api.UpdateSchemaContext{
```

Comment:
Like the comment in createSchemaUpdateIssue, we should also comment here about the database name pattern we support and what pipeline it leads to create